assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -486,6 +494,10 @@ describe("DiversionPage", () => {
       routeSegments: routeSegmentsFactory.build(),
     })
 
+    const intersection = "Avenue 1 & Street 2"
+    ;(useNearestIntersection as jest.Mock)
+        .mockReturnValueOnce({ ok: intersection })
```

Comment:
This test is failing because of `mockReturnValueOnce` - the hook is called a fairly large number of times, but after the first call, it's returning `null`, which is why the test is failing.

Also, and I'm aware that our current styles are inconsistent and nobody mentioned (until now 😇), but we're preferring to move towards the `jest.mocked` style of mocks, rather than `;(...)`. 

```suggestion
jest.mocked(useNearestIntersection).mockReturnValue({
  ok: intersection,
})
```

Reply:
Ohhhhhhhhhhhhhhhhhhh!!!!! Ahh!!! Thank you so much, this just clears all the fog right up